packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -47,7 +94,8 @@ const ApplicationDetails = () => {
     if (!data) {
       return;
     }
-    reset(data);
+    const formData = mapApplicationDataToFormData(data);
+    reset(formData);
```

Comment:
Better keep formData struct as ui designs. Instead, handle the data formating at the API layer or onSubmit handler.